fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
```

Comment:
The kubernetes metadata plugin likes JSON logs because it (me) likes structured logs (events). Anything logged in JSON will be merged into the fluentd record so that fields will show up in Elasticsearch as separate fields rather than just a single string.

If you want to disable it add `merge_json_log false` to the kubernetes filter plugin config.